conf/mesos-site.xml

```diff
@@ -19,55 +19,85 @@
 <configuration>
   <property>
     <name>mesos.hdfs.data.dir</name>
+    <description>The primary data directory in HDFS</description>
     <value>/tmp/hdfs/data</value>
   </property>
   
   <property>
     <name>mesos.hdfs.secondary.data.dir</name>
+    <description>The secondary data directory in HDFS</description>
     <value>/var/run/hadoop-hdfs</value>
   </property>
-  
+
+  <property>
+    <name>mesos.hdfs.native-hadoop-binaries</name>
+    <description>Mark true if you have hadoop pre-installed on your host machines (otherwise it will be distributed by the scheduler)</description>
+    <value>false</value>
+  </property>
+
   <property>
     <name>mesos.hdfs.framework.mnt.path</name>
+    <description>Mount location (if mesos.hdfs.native-hadoop-binaries is marked false)</description>
     <value>/opt/mesosphere</value>
   </property>
 
   <property>
     <name>mesos.hdfs.state.zk</name>
+    <description>Comma-separated hostname-port pairs of zookeeper node locations for HDFS framework state information</description>
     <value>localhost:2181</value>
   </property>
 
   <property>
     <name>mesos.master.uri</name>
+    <description>Zookeeper entry for mesos master location</description>
     <value>zk://localhost:2181/mesos</value>
   </property>
 
   <property>
     <name>mesos.hdfs.zkfc.ha.zookeeper.quorum</name>
+    <description>Comma-separated list of zookeeper hostname-port pairs for HDFS HA features</description>
     <value>localhost:2181</value>
   </property>
 
   <property>
     <name>mesos.hdfs.framework.name</name>
+    <description>Your Mesos framework name and cluster name when accessing files (hdfs://YOUR_NAME)</description>
     <value>hdfs</value>
   </property>
 
   <property>
     <name>mesos.hdfs.mesosdns</name>
+    <description>Whether to use Mesos DNS for service discovery within HDFS</description>
     <value>false</value>
   </property>
 
   <property>
     <name>mesos.hdfs.mesosdns.domain</name>
+    <description>Root domain name of Mesos DNS (usually 'mesos')</description>
     <value>mesos</value>
   </property>
 
   <property>
     <name>mesos.native.library</name>
+    <description>Location of libmesos.so</description>
     <value>/usr/local/lib/libmesos.so</value>
   </property>
 
   <property>
+    <name>mesos.hdfs.journalnode.count</name>
+    <description>Number of journal nodes (must be odd)</description>
+    <value>3</value>
+  </property>
+
+  <!-- Additional settings for fine-tuning -->
```

Comment:
Oh I misread this due to GH formatting, looking good :) I see they are all there.